Cargo.lock

```diff
@@ -2798,9 +2798,9 @@ checksum = "dc375e1527247fe1a97d8b7156678dfe7c1af2fc075c9a4db3690ecd2a148068"
 
 [[package]]
 name = "proc-macro2"
-version = "1.0.51"
+version = "1.0.56"
```

Comment:
Issue: If proc-macro2 1.0.56 is required now, that should go into the Cargo.toml file, too

Reply:
it's "required" for warning-free code, which to me is not that huge a deal

Reply:
If it's only a warning than this is fine. My concern was if this would _break_ people who are using older locked crate versions (such as GN which has its own lockfile).

Reply:
To require it, please update Cargo.toml, not Cargo.lock

Reply:
Huh I wonder which branch I pushed this to...